outlook/src/taskpane/api.js

```diff
@@ -1,15 +1,13 @@
 const api = {
     baseURL: localStorage.getItem('baseURL'),
-    //createLead: "/mail_client_extension/lead/create",
-    getLeads: "/mail_client_extension/lead/get_by_partner_id",
-    //deleteLead: "/mail_client_extension/lead/delete",
+    createLead: "/mail_client_extension/lead/create",
+    createPartner: '/mail_client_extension/partner/create',
+    createTicket: "/mail_client_extension/ticket/create",
+    enrichCompany: "/mail_client_extension/partner/enrich_and_create_company",
     getPartner: "/mail_client_extension/partner/get",
-    logMail: "/mail_client_extension/log_single_mail_content",
-    getInstalledModules: "/mail_client_extension/modules/get",
-    contactCreate: '/mail_client_extension/partner/create',
-    redirectCreateLead: "/mail_client_extension/lead/create_from_partner",
-    iapLeadEnrichmentEmail: "https://iap-services.odoo.com/iap/clearbit/1/lead_enrichment_email", // TODO remove
     iapLeadEnrichment: "https://iap-services.odoo.com/iap/mail_extension/enrich",
+    logSingleMail:"/mail_client_extension/log_mail_content",
+    queryPartners: "/mail_client_extension/partners/search",
```

Comment:
searchPartner ?

by the way: /mail_client_extension/partners/search -> should remove the "s" from partners to keep a coherent namespacing

Reply:
Done,

The odoo route definition has also been changed.